server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -3684,14 +3683,9 @@ public void testAbortedSnapshotDuringInitDoesNotStart() throws Exception {
 
             // The deletion must set the snapshot in the ABORTED state
             assertBusy(() -> {
-                try {
-                    SnapshotsStatusResponse status =
-                        client.admin().cluster().prepareSnapshotStatus("repository").setSnapshots("snap").get();
-                    assertThat(status.getSnapshots().iterator().next().getState(), equalTo(State.ABORTED));
-                } catch (Exception e) {
```

Comment:
This isn't necessary anymore, we'll never create a broken repository with this fix.